lib/sycamore/sycamore/transforms/embed.py

```diff
@@ -98,7 +98,8 @@ def generate_embeddings(self, doc_batch: list[Document]) -> list[Document]:
         assert self._transformer is not None
 
         text_batch = [self.pre_process_document(doc) for doc in doc_batch if doc.text_representation is not None]
-        embeddings = self._transformer.encode(text_batch, batch_size=self.model_batch_size, device=self.device)
+        if len(text_batch) > 0:
+            embeddings = self._transformer.encode(text_batch, batch_size=self.model_batch_size, device=self.device)
```

Comment:
slightly prefer 'if len(text_batch) == 0 then return early' just so I don't have to think through the case where `embeddings` is unbound